### PR TITLE
reconstruction of the page with a new library

### PR DESCRIPTION
**Pull request by dandelionadia**

### CODE REVIEW

File: src/components/HomeInfo.js [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
     render() {
         return (
             <div className="container-homeInfo">
                 <h3>subtitle</h3>
-                <h1>title to tilorem</h1>
+                <Title fontSize={45} fontSizeMd={70}>title to tilorem</Title>
                 <div className="container-messager">
                     <Avatar />
-                    <span className="message">{this.props.children}</span>
+                    <StyleMessage className="message">{this.props.children}</StyleMessage>
```
Comment (kettanaito) [inline src/components/HomeInfo.js]: Why do you need to have a `message` class name?

File: src/pages/HomePage.js [hunk 2]
```diff
@@ -20,24 +20,29 @@ const areasTablet = `
...
+`
 
 const HomePage = () => (
     <Composition
         areas={areasMobile}
         areasMd={areasTablet}
         gutter={2}
         padding={2}
+        templateColsMd="40% auto"
+        gap={2}
```
Comment (kettanaito) [inline src/pages/HomePage.js]: `gap` is the same as `gutter`. Please remove the `gap` prop.

Could you please point me to where have you found the `gap` prop being used? It's somewhere from the documentation, right?

File: src/pages/HomePage.js [hunk 3]
```diff
@@ -20,24 +20,29 @@ const areasTablet = `
...
-                <Header>
+                <Header as={StyleHeader}>
                     <Logo />
                     <Menu />
                 </Header>
                 <Image>
                     <img className="background__img" src="https://cdn.dribbble.com/users/1355613/screenshots/6568571/____41.jpg"></img>
                 </Image>
-                <Info>
+                <Info flex justifyContent='center'>
```
Comment (kettanaito) [inline src/pages/HomePage.js]:
```diff
- justifyContent='center'
+ justifyContent="center"
```

File: src/pages/HomePage.js (hunk 2 above)
Comment (dandelionadia) [reply src/pages/HomePage.js]: 'gap' is right =) 'gutter' is wront :P

File: src/components/HomeInfo.js (hunk 1 above)
Comment (dandelionadia) [reply src/components/HomeInfo.js]: because I need   color: $color-grey;
I will change it later

File: src/components/HomeInfo.js [hunk 4]
```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import { Button } from '../atoms/Button';
 import { Avatar } from '../atoms/Avatar';
 import styled from 'styled-components';
-import { useResponsiveProps } from 'atomic-layout'
+import { useResponsiveProps } from 'atomic-layout';
+import { Box } from 'atomic-layout';
```
Comment (kettanaito) [inline src/components/HomeInfo.js]: You can combine imports from the same module (library):

```js
import { A, b, c } from 'package-name'
```

File: src/components/Menu.js [hunk 5]
```diff
@@ -35,15 +37,13 @@ class Menu extends React.Component {
...
-                        <li key={index} className="menu__item">
-                            <a className="menu__link" href="#">{item.value}</a>
-                        </li>
-                    ))}
-                </StyleMenu>
-            </div>
+            <StyleMenu>
+                {menu.map((item, index) => (
+                    <li key={index} className="menu__item">
+                        <a className="menu__link" href="#">{item.value}</a>
```
Comment (kettanaito) [inline src/components/Menu.js]: I recommend to put a menu item's URL into the data structure:

```jsx
const menu = [
  { value: 'foo', url: '#' },
  ...
]

...

<a className="menu__link" href={item.url}>{item.value}</a>
```

This way _all_ the data about your menu is in the JSON (array).

**Pull request merged**